samples/topics.js

```diff
@@ -175,6 +175,60 @@ async function publishBatchedMessages(
   // [END pubsub_publisher_batch_settings]
 }
 
+async function publishWithRetrySettings(projectId, topicName, data) {
+  // [START pubsub_publisher_retry_settings]
+  // Imports the Google Cloud client library
+  const PubSub = require('@google-cloud/pubsub');
```

Comment:
The new syntax didn't work for me when I tried to add retry settings. So I followed the documentation here: https://cloud.google.com/nodejs/docs/reference/pubsub/0.20.x/v1.PublisherClient#publish

Reply:
The new way of making a publisher won't allow me to modify retry settings: https://cloud.google.com/nodejs/docs/reference/pubsub/0.20.x/Publisher#publish

Reply:
I see, you're using the underlying gapic code.

Let's modify it to
```
const {v1} = require('@google-cloud/pubsub');
```
so we won't confuse it with other samples where `PubSub` refers to the `class`, instead of the module :)

Reply:
This is super helpful!